lab_2_tokenize_by_bpe/main.py

```diff
@@ -47,6 +97,32 @@ def merge_tokens(
     :param pair: a pair of tokens to be merged
     :return: dictionary in the form of <preprocessed word: number of occurrences>
     """
+    if not(
+        isinstance(word_frequencies, dict)
+        and isinstance(pair, tuple)
+    ):
+        return None
+
+    merged_dict = {}
+
+    for word_tokens in word_frequencies:
+        word_tokens_list = list(word_tokens)
+        if pair[1] in word_tokens:
+            pair_token_index = []
+            for index in range(len(word_tokens) - 1):
+                token_pair = (word_tokens[index], word_tokens[index + 1])
+                if token_pair == pair:
+                    pair_token_index.append(index)
+
+            for pair_index in pair_token_index:
```

Comment:
Congratulations on solving the secret! But there are some mistakes. Think about this line. It is in this place that you have problems and you get "ве" instead of "ле". Maybe debugging can be useful for you! Good luck!

Reply:
fixed it!

Reply:
<img width="1276" alt="image" src="https://github.com/fipl-hse/2023-2-level-labs/assets/116805511/fbf19a93-b9af-44a8-b76f-0f0b245fc164">